src/content/blog/node-test-migration.mdx

```diff
@@ -0,0 +1,133 @@
+---
+title: "Migrating 500+ tests from Mocha to Node.js"
+description: "A retrospective of how Astro migrated more than 500 test suites from Mocha to Node.js test runner."
+publishDate: "March 21, 2024"
+authors:
+  - ema
+  - bjorn
+lang: "en"
+coverImage: "/src/content/blog/_images/test-migration/og_astro_test_migration.webp"
+socialImage: "/src/content/blog/_images/test-migration/header_astro_test_migration.webp"
+---
+
+Over a month ago, we discussed a possible migration to the Node.js test runner. While we were sufficiently happy with Mocha, we are always looking to make our CI jobs faster.
+
+The prospective move to relying on a test runner baked inside our runtime had some advantages for our main monorepo:
+- two fewer dependencies to install and maintain in our monorepo: `mocha` and `chai`;
+- maintainability: there are more people involved in the Node.js project to maintain the Node.js test runner;
+- future benefits: we believe that the test runner will only get better and better, and eventually save some time in our CI workflows.
```

Comment:
Bringing back this suggestion that kinda got lost in limbo because of Git confusion.
```suggestion
Relying on a test runner baked inside our runtime had some advantages for our main monorepo:
- Two fewer dependencies to install and maintain in our monorepo: `mocha` and `chai`.
- Maintainability: there are more people involved in the Node.js project to maintain the Node.js test runner.
- Future benefits: we believe the test runner will improve with time, and eventually save some time in our CI workflows.
```